config/locales/en.yml

```diff
@@ -595,6 +595,40 @@ en:
           empty_search:
             title: No invoices found
             primary: Clear all filters
+      settings:
+        show:
+          switch_button: Switch to %{mode}
+          switch_confirm: Are you sure you want to switch to %{mode} mode?
+          postpaid_description_html:
+            <p>
+              Current mode is <b>Postpaid</b>.
+            </p>
+            <p>
+              In postpaid mode, all fixed fees as well as variable fees are billed at the end of the
+              month.
+            </p>
+          prepaid_description_html:
+            <p>
+              Current mode is <b>Prepaid</b>.
+            </p>
+            <p>
+              In prepaid mode, all fixed fees and setup fees are billed immediately at the beginning
+              of the month or at the beginning of any pro-rated billing period. Variable costs are
+              always calculated at the end of the month.
+            </p>
+          deprecation_warning:
+            The %{gateway} gateway has been deprecated. It can still be used but you won't be able
+            to switch back to it once you confirmed the switch to another gateway.
+          save_gateway_confirm_1:
+            Changing your payment gateway will make all your buyers' credit card details invalid, so
+            they will have to enter the credit card information again to be billed next month.\n\n
+            Are you sure you want to change your payment gateway?
+          save_gateway_confirm_2:
+            Are you sure? If you click OK, all credit card details from all your customers will be
```

Comment:
Some weird characters in here:
![image](https://github.com/3scale/porta/assets/1270328/7ea56916-956b-4728-b264-b9f1501051b5)

Reply:
And the 1st confirm also

Reply:
Good catch 👍🏻